unconstrained_test.go

```diff
@@ -1503,16 +623,5 @@ func testMinimize(t *testing.T, tests []UnconstrainedTest, method Method) {
 				}
 			}
 		}
-
```

Comment:
The starting location is a slightly perturbed minimum that does not pass the initial convergence check. Linesearch returns "failed to converge" error on the first iteration.